src/bin/combine.rs

```diff
@@ -19,19 +23,24 @@ fn argparse<'a>() -> ArgMatches<'a> {
 }
 
 fn main() {
+    // Init env_logger
+    env_logger::init().expect("Failed to initiate logger");
+
     let _ = argparse();
 
     // Read each line
     let mut input_file = std::io::stdin();
     let mut shares_string = String::new();
     input_file
         .read_to_string(&mut shares_string)
-        .unwrap_or_else(|err| panic!("Error while reading stdin: {}", err));
+        .unwrap_or_else(|err| { error!("Error while reading stdin: {}", err);
+                                exit(1)});
```

Comment:
Little nit: I think this would me more readable if we inserted an extra newline:

```rust
        .unwrap_or_else(|err| {
            error!("Error while reading stdin: {}", err);
            exit(1)
        });
```